rerun_py/rerun_sdk/rerun/archetypes/transform3d_ext.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from typing import Any
+
+from rerun.datatypes.mat3x3 import Mat3x3Like
+from rerun.datatypes.rotation3d import Rotation3DLike
+from rerun.datatypes.scale3d import Scale3DLike
+from rerun.datatypes.transform3d import Transform3DLike
+from rerun.datatypes.translation_and_mat3x3 import TranslationAndMat3x3
+from rerun.datatypes.translation_rotation_scale3d import TranslationRotationScale3D
+from rerun.datatypes.vec3d import Vec3DLike
+
+
+class Transform3DExt:
+    def __init__(
+        self: Any,
+        transform: Transform3DLike | None = None,
+        *,
+        translation: Vec3DLike | None = None,
+        rotation: Rotation3DLike | None = None,
+        scale: Scale3DLike | None = None,
+        matrix: Mat3x3Like | None = None,
```

Comment:
I think we should call this `mat3x3`, otherwise people will try `Transform3D(matrix=my4x4matrix)` and expect it to work.

Same for the `TranslationAndMat3x3` constructor probably

Reply:
Agreed fro the `TranslationAndMat3x3`, but on `Transform3D` I want to use the same parameter in the future for 4x4 🤔